readme_renderer/rst.py

```diff
@@ -14,20 +14,28 @@
 from __future__ import absolute_import, division, print_function
 
 import io
+import typing
 
 from docutils.core import publish_parts
+from docutils.nodes import colspec, image
 from docutils.writers.html4css1 import HTMLTranslator, Writer
 from docutils.utils import SystemMessage
 
 from .clean import clean
 
 
-class ReadMeHTMLTranslator(HTMLTranslator):
+class ReadMeHTMLTranslator(HTMLTranslator):  # type: ignore
```

Comment:
When using `type: ignore`, it's best practice to indicate why. That could include ignoring a specific error, and/or a reference to a related GitHub issue. See the [mypy docs](https://mypy.readthedocs.io/en/stable/type_inference_and_annotations.html#silencing-type-errors) for details, and an [elaborate example in Twine](https://github.com/pypa/twine/blob/a9e9cd6ecec3c4d4504704ccaba08be487d67ebf/twine/package.py#L127).

Same thing applies to `object_image_types` below.

Reply:
For this line, the parent class is returning Any, and strict mode disallowed that.  I'll clarify the ignore comment

For the other line, I left a lot of detail in the commit message https://github.com/pypa/readme_renderer/pull/225/commits/a0903b16a59c73e3456906d7060b0f79c6a7f802 - I didn't want to pollute the code much with commentary about types.

Is that approach insufficient?